src/Camera/DefaultWorldCameraModel.cpp

```diff
@@ -214,6 +214,7 @@ bool DefaultWorldCameraModel::ConstrainCamera(std::chrono::microseconds dt, floa
 
 bool DefaultWorldCameraModel::ConstrainAltitude()
 {
+	return false;
```

Comment:
@bwrsandman Open to ideas on how to toggle this constraint for the citadel interior

Reply:
`Locator::config`